content/html/power-request-ci.html

```diff
@@ -4,21 +4,19 @@
   <div class="field field-name-body field-type-text-with-summary field-label-hidden">
     <div class="field-items">
       <div class="field-item even" property="content:encoded">
-        <p>The Open Source Lab partners with IBM to host POWER based servers in
-        order to provide an open platform for innovation to the open source
-        community. <b>For more information about this partnership, please visit
-        our <a href="/services/powerdev/">POWER Development Hosting
-        page</a></b>.
-        <p>Please use the form below to request access to the POWER CI service
-        provided by the OSUOSL.</p>
-        <p>This access is intended only for <b><i>free and open source</i></b>
-        projects who qualify and are approved by the OSUOSL. For proprietary
-        sourced projects or products please use other POWER resources which can
-        be found <a
-        href=https://developer.ibm.com/linuxonpower/cloud-resources/>here</a>.
-        The POWER CI service uses Docker to deploy Jenkins workers. In the
-        future we will add access to use OpenStack to deploy Jenkins workers
-        (ppc64le only).</p>
+        <p>The Open Source Lab partners with IBM to host POWER based servers in order to provide an open platform for
+          innovation to the open source community. <b>For more information about this partnership, please visit our
+          <a href="/services/powerdev/">POWER Development Hosting page</a></b>.</p>
+        <p>Please use the form below to request <b>new</b> access to the POWER CI service provided by the OSUOSL.</p>
+        <p>For any <b>changes, updates, or issues with an existing project</b> please send the specific request as an
+          email to <a href="mailto:powerci-request@osuosl.org">powerci-request@osuosl.org</a> and include the original
+          project name in the subject line and the IP address(es) for the VM(s) in the message body of the request which
+          will help us narrow down what specific VM it's for.</p>
```

Comment:
VM's and IP addresses don't apply for these requests, instead please have them provide the name of the Jenkins job that has the issue.

Reply:
92fce37